Update metrics-definitions.md

wiersgallak opened the pull request:
<!-- Thank you for contributing a pull request! Here are a few tips to help you:

1. Learn how to update the handbook: https://handbook.mattermost.com/company/how-to-guides-for-staff/how-to-update-handbook
2. If this is your first contribution, please sign the Contributor License Agreement: http://www.mattermost.org/mattermost-contributor-agreement/
   - Once signed, you will be added to the Mattermost Approved Contributor List: https://docs.google.com/spreadsheets/d/1NTCeG-iL_VS9bFqtmHSfwETo5f-8MQ7oMDE5IUYJi_Y/pubhtml?gid=0&single=true
   - If you included your mailing address, you may receive a Limited Edition Mattermost Mug as a thank you gift after your pull request is merged: https://forum.mattermost.org/t/limited-edition-mattermost-mugs/143
-->

#### Summary
Updated MAU based on this conversation https://community-daily.mattermost.com/private-core/pl/jrjwd76mobne7e6baxtsmg1b8r of how reactivated and deactivating users in Mattermost affects the MAU statistic.

wiersgallak committed:
Update metrics-definitions.md

justinegeffen commented on operations/business-operations/analytics/metrics-definitions.md:
```diff
@@ -396,6 +396,8 @@ Monthly active users are categorized into **engagement lifecycle segments**. The
 
 Currently only a subset of possible events, dubbed "whitelist" events, count towards MAU. The reason only a subset of events are captured is a result of exceeding Segment's \(third-party event logging service\) row limit. In the future, event logging will transition to rudder \(another third-party event logging service\), and the list of events that count towards MAU will become more expansive.
 
+Deactivating a user in Mattermost will result in MAU decreasing, as deactivated users are filtered out of the statistic's query.  However, reactivating a user in Mattermost will increase the MAU statistic. 
```
```suggestion
Deactivating a user in Mattermost will result in MAU decreasing, as deactivated users are filtered out of the statistic's query. However, reactivating a user in Mattermost will increase the MAU statistic. 
```

wiersgallak committed:
Update operations/business-operations/analytics/metrics-definitions.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

jasonblais commented:
@wiersgallak is this one ready to merge, or should we get Justine and Schiff do a second review?

wiersgallak commented:
I think @enelson720 still needs to review.  Here were some notes he provided me when we chatted via MM: 

> I think you have it added to the incorrect section. MAU is the user event-based MAU metric. active_users_montly,  active_users_daily, and active_user_count measures are from security and activity diagnostic telemetry and those are the measures impacted by deactivation. I also acknowledge that there are way too many variations of active users

justinegeffen commented on operations/business-operations/analytics/metrics-definitions.md:
```diff
@@ -396,6 +396,8 @@ Monthly active users are categorized into **engagement lifecycle segments**. The
 
 Currently only a subset of possible events, dubbed "whitelist" events, count towards MAU. The reason only a subset of events are captured is a result of exceeding Segment's \(third-party event logging service\) row limit. In the future, event logging will transition to rudder \(another third-party event logging service\), and the list of events that count towards MAU will become more expansive.
```
```suggestion
Currently only a subset of possible events, dubbed "whitelist" events, count towards MAU. The reason only a subset of events are captured is a result of exceeding Segment's \(third-party event logging service\) row limit. In the future, event logging will transition to Rudder \(another third-party event logging service\), and the list of events that count towards MAU will become more expansive.
```

justinegeffen commented on operations/business-operations/analytics/metrics-definitions.md:
```diff
@@ -407,4 +409,3 @@ Currently only a subset of possible events, dubbed "whitelist" events, count tow
 **Enterprise Trial Requests**
 
 * Number of trial license requests via [https://mattermost.com/trial](https://mattermost.com/trial) from Named Accounts or Enterprises with 5,000+ employees, in America, EMEA, Australia or Japan.
```
```suggestion
* Number of trial license requests via [https://mattermost.com/trial](https://mattermost.com/trial) from Named Accounts or Enterprises with 5,000+ employees, in America, EMEA, Australia, or Japan.
```

wiersgallak committed:
Update operations/business-operations/analytics/metrics-definitions.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

wiersgallak committed:
Update operations/business-operations/analytics/metrics-definitions.md

Co-authored-by: Justine Geffen <justinegeffen@users.noreply.github.com>

justinegeffen commented:
@enelson720, following up on this to confirm whether it's ready to merge.

justinegeffen commented on operations/business-operations/analytics/metrics-definitions.md:
```diff
@@ -394,7 +394,9 @@ Monthly active users are categorized into **engagement lifecycle segments**. The
-Currently only a subset of possible events, dubbed "whitelist" events, count towards MAU. The reason only a subset of events are captured is a result of exceeding Segment's \(third-party event logging service\) row limit. In the future, event logging will transition to rudder \(another third-party event logging service\), and the list of events that count towards MAU will become more expansive.
+Currently only a subset of possible events, dubbed "whitelist" events, count towards MAU. The reason only a subset of events are captured is a result of exceeding Segment's \(third-party event logging service\) row limit. In the future, event logging will transition to Rudder \(another third-party event logging service\), and the list of events that count towards MAU will become more expansive.
+
+Deactivating a user in Mattermost will result in MAU decreasing, as deactivated users are filtered out of the statistic's query. However, reactivating a user in Mattermost will increase the MAU statistic. 
```
```suggestion
Deactivating a user in Mattermost will result in MAU decreasing, as deactivated users are filtered out of the statistic's query. However, reactivating a user in Mattermost will increase the MAU statistic.
```

justinegeffen committed:
Update operations/business-operations/analytics/metrics-definitions.md

justinegeffen committed:
Merge branch '0.2.1' into MAU-Details

justinegeffen commented:
Gentle reminder, @enelson720 and @michaelschiffmm. :)

enelson720 commented:
> Gentle reminder, @enelson720 and @michaelschiffmm. :)

Sorry, just saw this. Reviewing now.

enelson720 reviewed (changes requested):
This portion of MAU is for user-level MAU events. The server-level diagnostics send `active_users_monthly` statistics that are impacted by the activating and deactivating of users, but this should not impact the user-level MAU calculation, so I don't believe this would be the correct place to have the callout. Perhaps moving it to the TEDAU section as the process of activating and deactivating users also impacts it as well. @justinegeffen

justinegeffen committed:
Update metrics-definitions.md

Punctuation formatting, removed extra white spaces. Moved MAU addition to TEDAU as suggested.

justinegeffen commented:
Thanks, @enelson720. I've moved the section as suggested.

One other note is that in some cases the dollar value is presented as "30k" and in others it's "$30k" (as an example). If you are able to review that and confirm whether that is correct or they should all have a $ sign that'd be great. :)

justinegeffen commented:
hey @enelson720. Just following up on this to find out whether we can merge or should close this PR. :)

enelson720 committed:
Merge branch '0.2.1' into MAU-Details

enelson720 reviewed (approved):
LGTM
